app/webpacker/controllers/internal_events_controller.js

```diff
@@ -0,0 +1,47 @@
+import { Controller } from 'stimulus';
+
+export default class extends Controller {
+  static targets = ['name', 'startAt', 'partialUrl'];
+
+  get nameValue() {
```

Comment:
I wouldn't bother with these getters, its just a layer of indirection it doesn't add anything IMO